### PR TITLE
ICS Nerva - Library, Command Dorms swapped location + rework, Stowaway spawns added.

### PR DESCRIPTION
**Pull request by Yantarii**
### Proposed Changes:
- Stowaways now have additional spawn points around specific hidden areas of maintenance.
- Moving Library above the Bar.
- Moving Command Dorms to previous Library area (which suits being near dorms.)
- Adjusts Bar to accommodate for an additional stairway.

### Bar:

Bar has been slightly compacted further upwards one tile to accommodate a stairwell.
Adjusts some of the electronics, and lights around the bar to accommodate for this change.
Moves kitchen sink to the top of the Bar.

Images:

### Library

Library has been moved to be above the Bar, allowing this area to be slightly more high-trafficked. This also should help facilitate additional roleplay oppertunities, i.e playing DnD with bar guests, etc. The Library has kept some of it's original damage as well, which can easily be repaired. There is a view down to the bottom level of the bar than can be used for all sorts of fun activities. The Library doesn't have direct line of sight to the teleporter area, meaning that traitors can still break into secure areas without too much difficulty, provided they take maintenance, or go to the north area of the library.

### Command Dorms:

Command Dorms have been moved near dorms itself, as it's seldom used.

### CODE REVIEW

File: maps/nerva/nerva-3.dmm [hunk 1]
```diff
@@ -5228,7 +5154,7 @@
 	input_tag = "d2_c_chamber_in";
 	name = "Chamber Gas Control";
 	output_tag = "d2_c_chamber_out";
-	sensors = list("d2_c_chamber_sensor"="Combustion Chamber")
+	sensors = list("d2_c_chamber_sensor"="Combustion                                Chamber")
```
Comment (CPU-Blanc) [inline maps/nerva/nerva-3.dmm:5157]: and again
```suggestion
	sensors = list("d2_c_chamber_sensor"="Combustion Chamber")
```

File: maps/nerva/nerva-3.dmm [hunk 2]
```diff
@@ -8014,7 +7817,7 @@
 	},
 /obj/structure/catwalk,
 /obj/machinery/navbeacon{
-	codes = list("patrol"=1,"next_patrol"="Deck 2 Aft Starboard");
+	codes = list("patrol"=1,"next_patrol"="Deck                                2                                Aft                                Starboard");
```
Comment (CPU-Blanc) [inline maps/nerva/nerva-3.dmm:7820]: as above
```suggestion
	codes = list("patrol"=1,"next_patrol"="Deck 2 Aft Starboard");
```

File: maps/nerva/nerva-3.dmm [hunk 3]
```diff
@@ -14620,11 +14512,19 @@
 	dir = 4
 	},
 /obj/machinery/navbeacon{
-	codes = list("patrol"=1,"next_patrol"="Deck 2 Fore Port");
+	codes = list("patrol"=1,"next_patrol"="Deck                                2                                Fore                                Port");
```
Comment (CPU-Blanc) [inline maps/nerva/nerva-3.dmm:14515]: another big boy
```suggestion
	codes = list("patrol"=1,"next_patrol"="Deck 2 Fore Port");
```

File: maps/nerva/nerva-3.dmm [hunk 4]
```diff
@@ -17280,18 +17405,40 @@
...
+/area/civilian/lounge)
 "VK" = (
 /obj/structure/cable{
 	d1 = 4;
 	d2 = 8;
 	icon_state = "4-8"
 	},
 /obj/machinery/navbeacon{
-	codes = list("patrol"=1,"next_patrol"="Deck 1 Port Docks");
+	codes = list("patrol"=1,"next_patrol"="Deck                                1                                Port                                Docks");
```
Comment (CPU-Blanc) [inline maps/nerva/nerva-3.dmm:17429]: long boy
```suggestion
	codes = list("patrol"=1,"next_patrol"="Deck 1 Port Docks");
```

File: maps/nerva/nerva-4.dmm [hunk 5]
```diff
@@ -340,7 +340,7 @@
 	icon_state = "intact"
 	},
 /obj/machinery/navbeacon{
-	codes = list("patrol"=1,"next_patrol"="Deck 1 Starboard Docks");
+	codes = list("patrol"=1,"next_patrol"="Deck  1  Starboard  Docks");
```
Comment (CPU-Blanc) [inline maps/nerva/nerva-4.dmm:343]: as above
```suggestion
	codes = list("patrol"=1,"next_patrol"="Deck 1 Starboard Docks");
```

File: maps/nerva/nerva-4.dmm [hunk 6]
```diff
@@ -9057,11 +9060,18 @@
 	},
 /obj/effect/catwalk_plated,
 /obj/machinery/navbeacon{
-	codes = list("patrol"=1,"next_patrol"="Deck 4 Escape");
+	codes = list("patrol"=1,"next_patrol"="Deck  4  Escape");
```
Comment (CPU-Blanc) [inline maps/nerva/nerva-4.dmm:9063]: again
```suggestion
	codes = list("patrol"=1,"next_patrol"="Deck 4 Escape");
```

File: maps/nerva/nerva-1.dmm [hunk 7]
```diff
@@ -2409,7 +2409,7 @@
 	input_tag = "d4_port_chamber_in";
 	name = "Chamber Gas Control";
 	output_tag = "d4_port_chamber_out";
-	sensors = list("d4_port_chamber_sensor"="Combustion Chamber")
+	sensors = list("d4_port_chamber_sensor"="Combustion  Chamber")
```
Comment (CPU-Blanc) [inline maps/nerva/nerva-1.dmm:2412]: added whitespace
```suggestion
	sensors = list("d4_port_chamber_sensor"="Combustion Chamber")
```

**Review by CPU-Blanc — CHANGES_REQUESTED** [review]
Only done a quick skim but there's some issues I've spotted.
Not sure why but it looks like your editor has added a bunch of whitespace into the beacon tags, which will break bot patrolling. Also a few sensors, which would break consoles properly linking to them

**Comment by Yantarii** [comment]
Weird, I'm using Visual Studio Code and StrongDMM. I'll fix them up in the next few days, thanks!

**Comment by Glloyd** [comment]
I like the change, I've been wanting to add a second level to the bar for a while. it might be best to recreate this on the baymerge branch however, this will be quite difficult to merge into there from master.

**Comment by Yantarii** [comment]
> I like the change, I've been wanting to add a second level to the bar for a while. it might be best to recreate this on the baymerge branch however, this will be quite difficult to merge into there from master.

Yeah I'm gonna close this and redo it for baymerge.

**Pull request closed without merging**